### PR TITLE
Uplift Kubernetes version to 1.22.3

### PR DESCRIPTION
**Pull request by Sunnatillo**
1.22.2 -> 1.22.3

### CODE REVIEW

File: vars.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ assured that they are persisted.
 | POD_CIDR | Pod CIDR | "x.x.x.x/x" | "192.168.0.0/18" |
 | NODE_HOSTNAME_FORMAT | Node hostname format. This is a format string that must contain exactly one %d format field that will be replaced with an integer representing the number of the node. | "node-%d" |
 | KUBERNETES_VERSION | Kubernetes version | "x.x.x" | "1.21.0" |
```
Comment (furkatgofurov7) [inline vars.md:52]: this should most probably be "v1.22.2"?

**Review by furkatgofurov7 — APPROVED** [review]
/approve

File: vars.md (hunk 1 above)
Comment (Sunnatillo) [reply vars.md:52]: it should be "v1.22.3"

**Pull request merged**